### PR TITLE
Permalink

### LINKED ISSUE #393: Add way to link specific territory
For example, lets say that I welcomed a new user ( like at https://www.openstreetmap.org/changeset/95848279#map=19/50.01753/19.86268&layers=N )

I want to link him list of places where (s)he may ask for help. It would be nice to have way to link to place on map with side panel already opened - right now I would need to add extra instructions about clicking on map to see useful things.

Right now I see no way to do it with https://openstreetmap.community/

### PR DESCRIPTION
**Pull request by kmpoppe**
Resolves #393

@bhousel I hope you don't mind my quite radical moving around code in the index.html to make this work :)

^Kai

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -58,6 +58,8 @@
 
 // define access token
 mapboxgl.accessToken = 'pk.eyJ1IjoibWlrZWxtYXJvbiIsImEiOiJjaWZlY25lZGQ2cTJjc2trbmdiZDdjYjllIn0.Wx1n0X7aeCQyDTnK6_mrGw';
+// cdn base address
+const cdnBase = 'https://cdn.jsdelivr.net/gh/osmlab/osm-community-index@main/dist/';
```
Comment (bhousel) [inline docs/index.html:62]: Need to stick to `var` to keep things working in IE11 without introducing a build step.

File: docs/index.html [hunk 2]
```diff
@@ -86,56 +87,94 @@
...
+      iZoom = parseInt(getParameterByName("zoom"));
+      if (isNaN(iZoom)) { iZoom = 7; }
+    }
+    let coords = getParameterByName("map").split(/\,/);
+    map.flyTo({center: [coords[1], coords[0]], zoom: iZoom, essential: true});
+  }
+
+});
+
+map.on('moveend', ({ originalEvent }) => {
```
Comment (bhousel) [inline docs/index.html:105]: I think fat arrows won't work in IE11 either.  Can use `function() { }` instead

File: docs/index.html [hunk 3]
```diff
@@ -86,56 +87,94 @@
...
+  if (originalEvent) {
+    // do nothing
+  } else {
+    map.fire('flyend');
+  }
+});
+
+map.on('flyend', e => {
```
Comment (bhousel) [inline docs/index.html:113]: same here

**Review by bhousel — APPROVED** [review]
Looks ok overall - I'd stick to ES5 JavaScript to keep it working in old browsers.  You can merge it if you want!

**Comment by kmpoppe** [comment]
@bhousel was the ES5 reference just a hint or did I change something that's not ES5 compatible? I had thought not, because I just moved some code around and used the methods that where already available through the `map` object?

^Kai

File: docs/index.html (hunk 1 above)
Comment (kmpoppe) [inline docs/index.html:62]:
```suggestion
var cdnBase = 'https://cdn.jsdelivr.net/gh/osmlab/osm-community-index@main/dist/';
```

File: docs/index.html (hunk 2 above)
Comment (kmpoppe) [inline docs/index.html:105]:
```suggestion
map.on('moveend', function (originalEvent) {
```

File: docs/index.html [hunk 4]
```diff
@@ -86,59 +87,97 @@
...
 
+map.on('moveend', function (originalEvent) {
+  if (originalEvent) {
+    // do nothing
+  } else {
+    map.fire('flyend');
+  }
+});
+
+map.on('flyend', e => {
```
Comment (kmpoppe) [inline docs/index.html:113]:
```suggestion
map.on('flyend', function(e) {
```

**Pull request merged**